sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/metadata.yaml

```diff
@@ -31,6 +34,7 @@ scheduling:
   date_partition_parameter: null
   parameters:
   - submission_date:DATE:{{ds}}
+  - sample_id:INT64:{{sample_id}}
```

Comment:
`sample_id` won't be a parameter during normal runs, and so shouldn't be specified here.